modules/extensions/src/terrain/terrain-effect.ts

```diff
@@ -97,8 +97,8 @@ export class TerrainEffect implements Effect {
     const {terrainDrawMode} = layer.state;
 
     return {
-      // @ts-expect-error
-      heightMap: this.heightMap?.getRenderFramebuffer(),
+      picking: {isActive: false},
```

Comment:
Unrelated change?

Reply:
It is to fix the ts error. Is it incorrect?